### PR TITLE
fix: imgix, github typo

### PR DESCRIPTION
**Pull request by aulneau**
### What this does

- This enables the use of imgix for production deployments (to serve up formats like webp when supported)
- Uses srcset to size images properly on mobile devices
- Fixes a typo in the github url
- I created a small rehype plugin to give us image sizes for local images. This gives us the ability to know the aspect ratio of our images and build it such that the page will never reflow when the image is loaded

### CODE REVIEW

File: src/components/feedback.tsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ export const FeedbackSection: React.FC<BoxProps> = props => {
         mt={space(['extra-loose', 'extra-loose', 'base-loose'])}
       >
         <Link
-          href={`https://github.com/blockstack/docs/edit/master/src/${editPage}.md`}
+          href={`https://github.com/blockstack/docs/edit/master/src/pages${editPage}.md`}
```
Comment (hstove) [inline src/components/feedback.tsx:133]: This doesn't need a trailing slash? The original looks like `editPage` wouldn't start with a slash.

File: src/components/mdx/md-contents.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ export const styleOverwrites = {
     pre: {
       // px: space(['none', 'none', 'extra-loose', 'extra-loose']),
     },
-    img: {
+    '.img': {
```
Comment (hstove) [inline src/components/mdx/md-contents.tsx:41]: Why does it need the `.`?

File: src/components/mdx/image.tsx [hunk 3]
```diff
@@ -18,27 +18,72 @@ const getUrl = pathname => {
...
+
+    const aspectRatio = getAspectRatio(dimensions);
+
+    // the content section will
+    const width = dimensions.width <= 720 ? dimensions.width : '100%';
+    return (
+      <Box width={width} maxWidth="100%" padding="0 !important" position="relative" className="img">
+        <Box height="0" paddingBottom={`${aspectRatio}%`} width="100%" />
+        <BaseImg position="absolute" top={0} left={0} {...props} />
+      </Box>
```
Comment (hstove) [inline src/components/mdx/image.tsx:85]: Slick :)

File: src/components/mdx/md-contents.tsx (hunk 2 above)
Comment (aulneau) [reply src/components/mdx/md-contents.tsx:41]: previously the spacing on images was by tag, but we want to treat the top level element as the spacing control, and with the aspect ratio changes, the top level element can sometimes be a `div`, so both have the className `img`.

File: src/components/feedback.tsx (hunk 1 above)
Comment (aulneau) [reply src/components/feedback.tsx:133]: the pathname is always starts with `/`

**Pull request merged**